lib/msal-node/src/client/PublicClientApplication.ts

```
@@ -201,6 +201,13 @@ export class PublicClientApplication
             // Wait for server to be listening
             const redirectUri = await this.waitForRedirectUri(loopbackClient);
 
+            if (request.redirectUri) {
+                this.logger.warning(
```

I would prefer we throw here if it's unsupported. And that validation should occur as early as possible so as to avoid setting up the loopback server for an invalid config

---

Changed to throw instead